rules/S6964/metadata.json

```
@@ -0,0 +1,24 @@
+{
+  "title": "Value type property of a model class should be nullable or annotated with the Required attribute to avoid under-posting",
```

I'd change this since Required is not the only attribute as mentioned earlier.
```suggestion
  "title": "The value type properties of a model class should be nullable or annotated with a model validation attribute to avoid under-posting",
```

---

This change still applies -> "The value type properties"

---

Or maybe: 
```suggestion
  "title": "The value type properties of a model class should be nullable or marked as "Required" to avoid under-posting"
```